app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/DataObject.kt

```diff
@@ -0,0 +1,6 @@
+package org.fossasia.susi.ai.rest.responses.susi
+
+class DataObject {
+
+    val data: List<Map<String, Any>> = ArrayList()
```

Comment:
The problem is `List<Map<String, Any>>`

Reply:
@iamareebjamal I cannot break this into a serialised pojo because there are variable number of fields and variable keys and responses

Reply:
First it was Datum, now it is Map of String and Any

Reply:
the responses are fetched based on the action type and for the particular action type an implementation is done, for the table type the skills can be variable like "list of batman enemies" or "barcelona team players" these have different keys and values corresponding to them.

Reply:
But the Any part is cast to one of the Class at the end, right?

Reply:
Class here referring to a pojo?

Reply:
Yes

Reply:
Any is used here because server may send an integer type data or a string type data, or a double and it depends to the skill made the type of response that will be sent by the server

Reply:
So how do we know what data to be cast into when we display it to the user

Reply:
The response on being parsed is converted to string type and then passed to the adapter, so I changed Any to string and tested the apk it still works fine